sdcm/nemesis.py

```diff
@@ -832,6 +832,26 @@ def disrupt_snapshot_operations(self):
         if result.stderr:
             self.tester.fail(result.stderr)
 
+    def disrupt_alter_encryption(self, keyspace="keyspace1", table="standard1"):
```

Comment:
I think we should support a decorator that checks if the feature available before selecting this nemesis.
This nemesis would be available only for enterprise scylla.

Reply:
Yes.